src/transform.ts

```diff
@@ -105,3 +110,26 @@ document.body.appendChild(style);
 
 module.exports = exportedTokens`;
 }
+
+function processSass(src: string): string {
+  let sassLoadPaths: string;
+  try {
+    sassLoadPaths = fs
+      .readFileSync(path.join(CACHE_FOLDER, SASS_LOAD_PATHS_CONFIG), 'utf8')
+      .trim();
+  } catch {
+    // File doesn't exist
+    sassLoadPaths = JSON.stringify([]);
+  }
+
+  return `const sass = require('sass');
+const sassSrc = ${JSON.stringify(src)};
+
+const result = sass.compileString(sassSrc, {
+  loadPaths: ${sassLoadPaths},
+});
```

Comment:
Actually, we can compile saas in the transformer to leverage caching feature #51 (now we haven't implemented yet, but we can prepare) instead of run time. CSS Modules (#40) might need to change also.
Do we have any other reason we need to convert `saas` to run time?

Reply:
We don't have other reason to convert `sass` to run time, so I move it out of the return string